internal/services/apimanagement/api_management_logger_resource.go

```diff
@@ -70,9 +70,39 @@ func resourceApiManagementLogger() *pluginsdk.Resource {
 
 						"connection_string": {
 							Type:         pluginsdk.TypeString,
-							Required:     true,
+							Optional:     true,
 							Sensitive:    true,
 							ValidateFunc: validation.StringIsNotEmpty,
+							AtLeastOneOf: []string{
+								"eventhub.0.connection_string",
+								"eventhub.0.endpoint_uri",
+							},
+							ConflictsWith: []string{
+								"eventhub.0.endpoint_uri",
+								"eventhub.0.client_id",
+							},
+						},
+						"endpoint_uri": {
+							Type:         pluginsdk.TypeString,
+							Optional:     true,
+							ValidateFunc: validation.StringIsNotEmpty,
+							AtLeastOneOf: []string{
+								"eventhub.0.connection_string",
+								"eventhub.0.endpoint_uri",
+							},
+							ConflictsWith: []string{
+								"eventhub.0.connection_string",
+							},
+							RequiredWith: []string{"eventhub.0.client_id"},
+						},
+						"client_id": {
+							Type:         pluginsdk.TypeString,
+							Optional:     true,
+							ValidateFunc: validation.StringIsNotEmpty,
```

Comment:
Add the proper validation in 
```suggestion
							ValidateFunc: validation.IsUUID,
```